apps/api-extractor/src/extractor/Extractor.ts

```diff
@@ -290,7 +290,6 @@ export class Extractor {
       const apiJsonFilename: string = path.join(outputFolder, packageBaseName + '.api.json');
 
       this._monitoredLogger.logVerbose('Writing: ' + apiJsonFilename);
-      fsx.mkdirsSync(path.dirname(apiJsonFilename));
```

Comment:
>fsx.mkdirsSync(path.dirname(apiJsonFilename)); [](start = 6, length = 46)

Why are you deleting this?

Reply:
Looks like `ApiJsonGenerator.writeJsonFile` uses the `ensureFolderExists` option.

---
In reply to: [198695184](https://github.com/Microsoft/web-build-tools/pull/727#discussion_r198695184) [](ancestors = 198695184)